addons/Joystick/lua/weapons/gmod_tool/stools/wire_joystick.lua

```diff
@@ -1,344 +1,412 @@
-TOOL.Tab			= "Wire"
-TOOL.Category		= "Input, Output"
-TOOL.Name			= "Joystick"
-TOOL.Command		= nil
-TOOL.ConfigName		= ""
+local gsToolModeOP = TOOL.Mode
+local gsToolPrefix = gsToolModeOP.."_"
+local gsToolLimits = gsToolModeOP:gsub("_multi", "").."s"
+local gsSentClasMK = "gmod_"..gsToolModeOP
+local MappingFxUID = "abcdefghijklmnopqrstuvwxyz1234567890ABCDEFGHIJKLMNOPQRSTUVWXYZ"
+
+TOOL.Tab        = gsToolModeOP:match("%w+"):gsub("^%l", string.upper)
```

Comment:
Please leave this at "Wire" and the below as "Joystick" as this string manipulation is unnecessarily complicated

Reply:
Reverted in 97acdb1cc9315998cdd73092fceeb1dfbf1f8afc